tools/tl_brute.php

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/php
+
+<?php
+
+use TelegramOSINT\Exception\TGException;
+use TelegramOSINT\MTSerialization\OwnImplementation\OwnDeserializer;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$exact = false;
+if($argc < 2){
+    die('Usage: ./tl_brute.php <tl_hex_node> <exact: 1|0>');
```

Comment:
Pls add "\n" at the end of the line. Otherwise, in bash, it would look like:
```
$ ./tl_brute.php
Usage: ./tl_brute.php <tl_hex_node> <exact: 1|0>$ 
```

Reply:
+